src/portalnet/protocol.rs

```diff
@@ -148,10 +168,24 @@ impl PortalnetProtocol {
             data_radius: portal_config.data_radius,
         };
 
+        let db_path = match env::var(TRIN_DB_ENV_VAR) {
+            Ok(val) => val,
+            Err(_) => panic!(
+                "Must supply target trin db path as environment variable, like:\n\
+                {}=\"/path\"",
+                TRIN_DB_ENV_VAR,
+            ),
+        };
```

Comment:
This is basically the bare minimum to get things running. Seems like this strategy should be improved - I'm open to any suggestions. If I remember correctly, the plan is to stick with rocksdb until it becomes obvious/painful enough that we need to move to a relational database.

Reply:
I mean, we're already putting stuff in `~/.trin`, last I remember. Maybe just keep it there as a default for a while, until we think of something better.